src/ipc/process.cpp

```diff
@@ -72,7 +72,7 @@ static bool ParseAddress(std::string& address,
                   struct sockaddr_un& addr,
                   std::string& error)
 {
-    if (address.compare(0, 4, "unix") == 0 && (address.size() == 4 || address[4] == ':')) {
+    if (address.starts_with("unix") && (address.size() == 4 || address[4] == ':')) {
```

Comment:
I think we can simplify this further:
```suggestion
    if (address == "unix" || address.starts_with("unix:")) {
```

Reply:
The suggestion has been accepted.